### PR TITLE
Add support for bookmark demod and bandwidth

### PR DESCRIPTION
**Pull request by jedi7**
* Add storing of modulation into bookmark file
* Add storing of bandwidth into bookmark file
 (stored as offset low/high cut freq of frequency)
* Bookmark editor updated

### CODE REVIEW

**Comment by jedi7** [comment]
@BatchDrake Not sure how to propagate correctly the bandwidth to the GUI :( 
Issues:
* wrong bandwidth is selected (in gui spinbox) when clicked from bookmark manager
* wrong bandwidth is selected (in gui spinbox) when clicked from waterfall and USB/LSB is used.

File: UIMediator/UIMediator.cpp [hunk 1]
```diff
@@ -1188,16 +1204,29 @@ UIMediator::onOpenBookmarkManager(void)
...
+  this->ui->spectrum->setCenterFreq(info.frequency);
   this->ui->spectrum->setLoFreq(0);
 
-  this->onFrequencyChanged(frequency);
+  if (!info.modulation.isEmpty()) {
+    this->ui->audioPanel->setDemod(AudioPanel::strToDemod(info.modulation.toStdString()));
+  }
+
+  if (info.bandwidth() != 0) {
+    this->ui->spectrum->setFilterBandwidth(info.bandwidth());
```
Comment (BatchDrake) [inline UIMediator/UIMediator.cpp:1217]: My bad, this actually updates the spectrum only. We should use `this->setBandwidth(unsigned int)` which is the proper mediator method that updates all UI widgets related to the filter bandwidth (including both the spectrum and the inspector panel).

**Comment by BatchDrake** [comment]
btw I include screenshots of the problem with buttons in the bookmark manager, just in case is not reproducible in all computers:

File: UIMediator/UIMediator.cpp (hunk 1 above)
Comment (jedi7) [reply UIMediator/UIMediator.cpp:1217]: fixed and works :)

**Comment by jedi7** [comment]
You can try change the size of the column (by mouse). Anyway do you know how to set it as fixed with proper width?

**Comment by jedi7** [comment]
last issue is on bw change from clicking to waterfall.
When USB/LSB is used, then the result bw is twice. Any Hints?
I know it happens because of MainSpectrum::onWfBandwidthChanged. But do not know how to properly fix it.

**Comment by BatchDrake** [comment]
Don't worry about the bandwidth, for LSB and USB signals it should display twice the highlighted bandwidth. I'm doing this in order to match AM and SSB cutoff frequencies.

**Comment by jedi7** [comment]
I found way how to do it :) Now it should be all ok and consistent.

**Comment by BatchDrake** [comment]
> You can try change the size of the column (by mouse). Anyway do you know how to set it as fixed with proper width?

Just verified this my self and there's some odd behavior when adjusting them by mouse. I believe I secrewed up implementing the ButtonDelegate and that one's on me, so don't worry about it, I'll look into this one later myself.

**Pull request merged**